Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:carbon as build
+
+# Create app directory
+WORKDIR /usr/src/pika
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY pika/package*.json ./
+
+# If you are building your code for development
```

Comment:
Actually this only builds it for production because of the `--only=production` flag, right?

Reply:
Yes, I think it just uses the production-ready versions of the dependencies